bot/cogs/games/trivia.py

```diff
@@ -0,0 +1,126 @@
+import asyncio
+import traceback
+
+import aiohttp
+
+from collections import defaultdict
+
+from discord import Color, Embed, Message
+from discord.ext.commands import Cog, Context, group
+
+from bot.core.bot import Bot
+from bot import config
+
+
+def convert_markdown(in_string: str) -> str:
+    """Replaces markdown with its intended symbol."""
+    pass_one = in_string.replace("&quot;", '"').replace("&#039;", "''").replace("\\", "")
+    pass_two = pass_one.replace("&ntilde;", "").replace("&aacute;", "").replace("&amp;", "and")
+    return pass_two
+
+
+class Trivia(Cog):
+    """The Cog that handles Trivia commands."""
+    def __init__(self, bot: Bot) -> None:
+        self.bot = bot
+        self.session = aiohttp.ClientSession()
+        self.players = defaultdict(lambda: defaultdict(list))
+
+    @group(aliases=["quiz"])
+    async def trivia(self, ctx: Context) -> None:
+        """The group for all the trivia commands."""
+        pass
+
+    @trivia.command()
+    async def start(self, ctx: Context, questions: int = 10) -> None:
+        try:
```

Comment:
```suggestion
```
Since the command is no longer WIP, this should be removed
(command should also be de-dented one level, but since I can't highlight the entire command on touchpad, i'm saying it here)